package.json

```diff
@@ -21,31 +21,33 @@
   "type": "module",
   "main": "./dist/index.cjs",
   "exports": {
-    "import": "./dist/index.js",
-    "require": "./dist/index.cjs"
+    ".": {
```

Comment:
Dual-emit CJS / ESM intent is now inferred from the `package.json` output fields (`main`, `module`, `exports/require/module/import/default`).

The particular syntax variant previous used in this package for `exports` (as an object but without an actual export path is not yet supported in `viteup` though it should be trivial to add support for it).